Quick-Start.md

```diff
@@ -0,0 +1,209 @@
+
+1. Install Kueue resources, namely Cluster Queue, Resource Flavor, and Local Queue:
+   - Visit [Kueue Resources configuration](https://github.com/project-codeflare/codeflare-sdk/blob/main/docs/setup-kueue.md)
+
+## Launch a Jupyter Notebook Instance
+
+After setting up the Data Science Cluster components, you can start using the Jupyter notebooks for your data science projects. Here’s how to launch a Jupyter notebook:
+
+1. Access the RHOAI Dashboard:
+   - Navigate to the Red Hat OpenShift AI dashboard within your OpenShift web console.
+2. Create a Data Science Project:
+   - Go to the Data Science Projects section from the dashboard menu.
+   - Click on `Create data science project` and follow the prompts to set up a new project.
+3. Launch Jupyter Workbench:
+   - Inside your newly created project, find and click on the "Create Workbench" button.
+   - On the Workbench creation page, select "Standard Data Science" from the list of available notebook images. This image will include common data science libraries and tools that you might need.
+   - Configure any additional settings such as compute resources or environment variables as needed and `Create workbench`
+4. Access Your Notebook:
+   - Once the workbench is ready, click on the provided link or button to open your Jupyter notebook.
+## Submit your first job
+
+We can now go ahead and submit our first distributed model training job to our cluster.
+
+This can be done from any python based environment, including a script or a jupyter notebook. For this guide, we'll assume you've selected the "Jupyter Data Science" from the list of available images on your notebook spawner page.
+
+### Clone the demo code
+
+Once your notebook environment is ready, in order to test our CodeFlare stack we will want to run through some of the demo notebooks provided by the CodeFlare community. So let's start by cloning their repo into our working environment.
+
+```bash
+git clone https://github.com/project-codeflare/codeflare-sdk
+cd codeflare-sdk
+```
+
+For further development guidelines and instructions on setting up your development environment for codeflare-sdk, please refer to the [CodeFlare SDK README](https://github.com/project-codeflare/codeflare-sdk?tab=readme-ov-file#development).
+
+### Run the Guided Demo Notebooks
+
+Get started with the guided demo notebooks for the CodeFlare-SDK by following these steps:
+
+1. Access Your Jupyter Notebook Server:
+2. Update Your Notebook with Access Token and Server Details:
+   - Retrieve your OpenShift access token by selecting your username in the console, choosing "Copy Login Command", and then "Display Token".
+   - Open your desired demo notebook from the `codeflare-sdk/demo-notebooks/guided-demos` directory.
+   - Update the notebook with your access token and server details and run the demos.
+
+## Cleaning up the AI Platform Install
```

Comment:
I think we should include both uninstallation scenarios - removal with delete-all-in-one and uninstallation through installed operators in openshift, just to be consistent.